.flake8

```diff
@@ -1,6 +1,7 @@
 [flake8]
 max-line-length = 88
 max_complexity = 10
+ignore = E203
```

Comment:
Why?

Reply:
black and flake8 have conflicts about white spaces before colons